CONTRIBUTING.md

```diff
@@ -0,0 +1,33 @@
+# How to become a contributor and submit your own code
+
+We'd love to accept your issues and (especially) patches! If you have found a
+bug you can submit an issue describing it and one of the repo maintainers will
+get back to you. Alternately you can submit a PR and the sections below outline
+the process.
+
+## Contributing a Patch
+
+1. Fork the desired repo, develop and test your code changes.
+1. Ensure that your code has an appropriate set of unit tests which all pass.
+1. Break your changes in to a series of logical self-contained commits.
```

Comment:
Typo `in to` (or is it not?)

Reply:
You mean it should be `into`?

Reply:
That's what I would do, but again, I'm not a good reference point ;-)

Reply:
Yeah, "into" would be more normal, if we're getting in splitting grammatical hairs ;)